koku/api/tags/queries.py

```diff
@@ -80,6 +70,9 @@ def __init__(self, parameters):
             parameters    (QueryParameters): parameter object for query
 
         """
+        if parameters.get_filter("time_scope_value") == "-10":
```

Comment:
The initial reason to get rid of 10/30 days is the tags summary table is keyed off of the billing period (so -1 and -2 / monthly are naturally supported). 

This complicates the logic a little, but we could do some date checking here. If 10/30 days ago is in last month, we could return tags for both months. I think the tag query handler would just have to have its own `def `_get_date_delta` method that returns the correct date ranges? 

The date comparison can be done using our date helper class: https://github.com/project-koku/koku/blob/4a1a08abeca0c7db845a55da4e7bf10c41b4ef97/koku/api/utils.py#L26

It already has convenience methods for n_days_ago and month start/stops.